src/reporter/App.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react'
+import ReactDOM from 'react-dom'
+import Tabs from 'react-bootstrap/Tabs'
+import Tab from 'react-bootstrap/Tab'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+import Nav from 'react-bootstrap/Nav'
+import Report from './Report'
+import './App.css'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import results from './results'
+
+const getDefaultEnv = (n) => Object.keys(results[n]).sort().reverse()[0]
```

Comment:
I don't know the scope of the project, but in my case I would add all the abstracted segregated functions in a helper?